src/commonMain/kotlin/com.monta.changelog/printer/slack/SlackChangeLogPrinter.kt

```diff
@@ -15,23 +15,37 @@ import io.ktor.http.contentType
 
 class SlackChangeLogPrinter(
     private val slackToken: String,
-    private val slackChannel: String,
+    private val slackChannel: String?,
```

Comment:
imo you could make this `private val slackChannels: Set<String>` and make the channels outside of the classic